docs/content/configuration/download.md

```diff
@@ -72,6 +72,8 @@ If a user has their `GOPROXY` environment variable set with a comma separated li
 
 The last block specifies that any module matching `github.com/pkg/*` (such as `github.com/pkg/errors`) will always redirect the `go` tool to https://gocenter.io. In this case, Athens will never persist the given module to its storage.
 
+_NOTE_: https://pkg.go.dev/path/?tab=doc#Match is used to parse the contents of `download` if you are working with a package that contains `-` then you should escape that character, e.g. `test\\-pkg`
```

Comment:
Hmm, if this is true, I'd imagine it's more of a bug. 

If I understand correctly, a module named `github.com/pkg/hello-world` must be written as `github.com/pkg/hello\\-world` in the Download File HCL? 

Will definitely give it a test soon

Reply:
Yeah, we encountered this today on Slack (gophers/#athens), I haven't tested myself but it seemed a reasonable explanation for what @BradErz was encountering.

Reply:
So i just tried to replicate this again for the past hour and now i can't replicate it at all..

I also added a test to check in the `download/mode.DownloadFile.Match()` but it also passed, maybe i was just being ultimatley special.

I will remove this `__NOTE__`